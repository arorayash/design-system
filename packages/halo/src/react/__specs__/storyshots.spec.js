import initStoryshots from '@storybook/addon-storyshots'

jest.mock('@pluralsight/ps-design-system-storybook-addon-center')

initStoryshots()
