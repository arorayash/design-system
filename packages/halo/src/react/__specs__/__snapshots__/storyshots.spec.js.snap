// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots appearance primary 1`] = `
<div
  test-decorator-center={true}
>
  <div>
    something in here
  </div>
</div>
`;
